creatorkit/Toolbox.md

```diff
@@ -0,0 +1,157 @@
+![Creator Logo](../creatorlogo.png)
+
+# Toolbox
+
+The toolbox is a reference page for you to use to set up your kit. Follow the steps listed in the project pages and refer back to this page for more detail on individual steps.
+
+## Hardware
+
+* Creator Kit
+* Ubuntu 14.04 64-bit PC/Laptop
+
+## Powering Ci40 + Clicker boards
+
+There are 2 ways to power your Ci40 board.
+
+* Using a power supply
+* Using a microUSB cable connected to the port next to the power supply port
+
+There are also 2 ways to power Clicker boards
+
+* 2 AAA Batteries
+* MicroUSB
+
+## Getting the code
+
+You can get pre-built releases or source code of the Creator Kit projects to work with. The Releases will allow you to set up a working system quickly without having to build any code. The source code will allow you to inspect/extend the projects and build the releases yourself.
+
+[Releases](http://github.com/CreatorKit)
+
+[Source Code](http://github.com/CreatorKit/manifest)
+
+See [building code from source](#building-code-from-source) for information on how to use the source code.
+
+### Connecting to Ci40 via Serial
+
+![serial](serialport.resized.jpg)
+
+The next step is to connect to your Ci40 using a serial terminal. Connect a microUSB-USB A cable to the port within the Mikrobus connector and into your PC. You can use any serial terminal program you like set to the standard 115200-N-8-1. The following is an example using miniterm in Ubuntu
+
+<code>$ sudo miniterm.py /dev/ttyUSB0 -b 115200</code>
```

Comment:
It would be good to add it in as per the usage instructions and to avoid this to be seen as a mistake